templates/zerver/help/desktop-app-install-guide.md

```diff
@@ -1,73 +1,119 @@
 
-1. Download [zulip-x.x.x-arch.nsis.7z][LR]  [*here arch = ia32 (32-bit), x64 (64-bit)*]
-2. Extract the zip wherever you want (e.g. a flash drive) and run the app from there
+1. Download [zulip-x.x.x-x64.nsis.7z][latest] for 64-bit desktops
+   (common), or [zulip-x.x.x-ia32.nsis.7z][latest] for 32-bit (rare).
+2. Copy the installer file to the machine you want to install the app
+   on, and run it there.
+3. Done! Run Zulip from the Start menu.
+4. The app will NOT update automatically. You can repeat these steps
+   to upgrade to future versions. <!-- TODO fact check -->
 
-## Linux
+## Installing on Linux
 
-**Ubuntu, Debian 8+ (deb package)**:
+### apt (recommended for Ubuntu or Debian 8+)
 
-1. Download [Zulip-x.x.x-amd64.deb][LR]
-2. Double click and install, or run `dpkg -i Zulip-x.x.x-amd64.deb` in the terminal
-3. Start the app with your app launcher or by running `zulip` in a terminal
-4. Done! The app will NOT update automatically, but you can still check for updates
-
-**Other distros (Fedora, CentOS, Arch Linux etc)** :
-
-1. Download [Zulip-x.x.x-x86_64.AppImage][LR]
-2. Make it executable using `chmod a+x Zulip-x.x.x-x86_64.AppImage`
-3. Start the app with your app launcher
-
-**You can also use `apt-get` (recommended)**:
-
-* First download our signing key to make sure the deb you download is correct:
+1. Set up the Zulip Desktop apt repository and its signing key, from a
+   terminal:
 
 ```
 sudo apt-key adv --keyserver pool.sks-keyservers.net --recv 69AD12704E71A4803DCA3A682424BE5AE9BD10D9
+echo "deb https://dl.bintray.com/zulip/debian/ stable main" | \
+  sudo tee -a /etc/apt/sources.list.d/zulip.list
 ```
 
-* Add the repo to your apt source list :
+2. Install the client, from a terminal:
 ```
-echo "deb https://dl.bintray.com/zulip/debian/ beta main" |
-  sudo tee -a /etc/apt/sources.list.d/zulip.list
+sudo apt update
+sudo apt install zulip
 ```
 
-* Now install the client :
+3. Done! Run Zulip from your app launcher, or with `zulip` from a
+   terminal.
+4. The app will be updated automatically to future versions when
+   you do a regular software update on your system, e.g. with `sudo
+   apt update && sudo apt upgrade`.
+
+### AppImage (recommended for all other distros)
+
+1. Download [Zulip-x.x.x-x86_64.AppImage][latest]
+2. Make the file executable, with `chmod a+x
+   Zulip-x.x.x-x86_64.AppImage` from a terminal.
+3. Done! No installer necessary; this file is the Zulip app.  Run it
+   from your app launcher, or from a terminal.
+3. The app will NOT update automatically. You can repeat these steps
+   to upgrade to future versions.
+
+<!-- TODO why dpkg? -->
+
+# Beta releases
+
+Get a peek at new features before they're released!  If you'd like to
+be among the first to get new features in the Zulip desktop app, and
+to give the Zulip developers feedback to help make each stable release
+the best it can be, then you might like the beta releases.
+
+## Installing on macOS, Windows, or Linux with AppImage
```

Comment:
@gnprice `with AppImage` should be `Pre/Beta-release`?

> ## Installing on macOS, Windows, or Linux with Beta release

Reply:
Hmm, I found it tricky to word this heading clearly and wasn't totally satisfied with the result, and apparently I was right not to be. :-)

The intent here is to mean "installing on macOS, or installing on Windows, or using AppImage to install on Linux"; the AppImage part is only meant to apply to Linux. There are separate instructions just below for Linux not with AppImage, and with apt instead.

It's a subheading of the "Beta releases" section -- so that part doesn't need to be repeated.

Maybe the headings could be

> ## Installing on macOS, Windows, or Linux/AppImage
>
> ## Installing on Linux/apt

though I don't love that either.

Logically they could just be "on Linux using apt" and "other", but that doesn't communicate right -- if someone's on Windows, they're going to want to just look for the heading that says "Windows" rather than do a tiny logic puzzle.